src/main/java/namor/template/stack/ArrayStack.java

```diff
@@ -0,0 +1,71 @@
+package namor.template.stack;
+
+import java.util.Arrays;
+
+// TODO: 20.03.17 [GUSb] A test class to check everything works, should be deleted with first finished task
+public class ArrayStack {
+
+    private int top;
+    private int size;
+    private int[] stack;
+
+    public ArrayStack(int size)
+    {
+        this.size = size;
+        stack = new int[size];
+    }
+
+    public ArrayStack()
+    {
+        this.size = 10;
+        stack = new int[size];
+    }
+
+    public boolean IsEmpty()
+    {
+        return top == 0;
+    }
+
+    public void push(int value)
+    {
+        if (top == stack.length) {
+            extendStack();
+        }
+        stack[top] = value;
+        top++;
+    }
+
+    public int stackSize()
```

Comment:
Why in this name you repeat word "stack". Your class called stack it's meaningful that this is a size of stack